src/libs/OptionsListUtils.js

```diff
@@ -428,14 +458,15 @@ function getOptions(reports, personalDetails, activeReportID, {
 
         const reportPersonalDetails = getPersonalDetailsForLogins(logins, personalDetails);
 
-        // Save the report in the map if this is a single participant so we can associate the reportID with the
-        // personal detail option later.
-        if (logins.length <= 1) {
+        // Save the report in the map if this is a single participant (with the exception of PolicyExpenseChat and
+        // default rooms) so we can associate the reportID with the personal detail option later.
+        if (logins.length <= 1 && !isPolicyExpenseChat && !isDefaultRoom) {
```

Comment:
I'm checking these flags to prevent a bug where the policy and policyExpense (workspace) chats were included twice in the Search Page

Reply:
What was the bug and can we maybe say a bit more about it? We added to the comment...

>(with the exception of PolicyExpenseChat and default rooms)

It is not easy to see why we are making this exception. If I saw it I would just assume whoever wrote this had a good reason and stay away from it 😂

Reply:
I was having duplicated workspace chats, but that's now fixed with the participants' change I made [here](https://github.com/Expensify/App/pull/7852/files/6ad97fb3d52ef67c3d11b4506f9d802d83e2161e#r815350015), also I thought the same was happening with default rooms but now with more context, I realized this is not the case. I'm going to revert this condition and the comment. Good catch and thanks for pointing that!